jubeatfesto.html

```diff
@@ -11,6 +11,114 @@
         <script type="text/javascript">
             window.addEventListener("load", function() {
                 new PatchContainer([
+                    new Patcher("jubeat.dll", "2021-12-15", [
```

Comment:
Sorry, these are all a bit new for my n-1 year rule!

Reply:
I think we all have different standards for n-1, y-1 lol
+ removed the `festo` changes

Reply:
It would be better if konami released a new jubeat version already :(

Reply:
Yes, indeed. I didn't know, but n-0 version is the same 3 months 😅